account/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+# Create your models here.
+
+class Account(models.Model):
+	first_name = models.CharField(max_length=30)
+	last_name = models.CharField(max_length=40)
+	email = models.EmailField(max_length=100)
+	password = models.CharField(max_length=100)
```

Comment:
= 기준으로 필드명 정렬 부탁 드립니다.
```suggestion
	first_name = models.CharField(max_length=30)
	last_name  = models.CharField(max_length=40)
	email      = models.EmailField(max_length=100)
	password   = models.CharField(max_length=100)
```